dll/win32/windowscodecs/wincodecs_private.h

```diff
@@ -23,22 +23,33 @@
 #include "wincodecsdk.h"
 
 #include "wine/debug.h"
-#include "wine/unicode.h"
 
-DEFINE_GUID(CLSID_WineTgaDecoder, 0xb11fc79a,0x67cc,0x43e6,0xa9,0xce,0xe3,0xd5,0x49,0x45,0xd3,0x04);
+#ifdef __REACTOS__
+#include <wine/library.h>
+#define wcsdup _wcsdup
```

Comment:
Ah, for this, you need instead to add to the targets link libraries of the windowscodecs dll, "oldname" (and remove this define)